prometheus_juju_exporter/config.py

```diff
@@ -0,0 +1,58 @@
+"""Configuration loader."""
+import confuse
+
+from prometheus_juju_exporter.logging import get_logger
+
+config = None
+
+
+class Config:
+    """Configuration for PrometheusJujuExporter."""
+
+    def __init__(self, args=None):
+        """Initialize the config class."""
+        self.logger = get_logger()
+        global config
+
+        if config:
+            self.config = config
+        else:
+            self.config = confuse.Configuration("PrometheusJujuExporter", __name__)
+            config = self.config
+
+        if args:
+            self.config.set_args(args, dots=True)
+
+        self.validate_config_options()
+
+    def validate_config_options(self):
```

Comment:
I'm not sure why we use this confuse library, but we can see what it offers.

Example of validation:
```python
template = {"servers": confuse.Sequence({"host": str, "port": confuse.Choice(range(0, 65536), default=80)})}
valid_config = config.get(template)
```

Reply:
This is much cleaner indeed. I've made the changes to reflect this.